pkg/models/secrets_name_transformer.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright © 2021 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package models
+
+type SecretsNameTransformer struct {
+	Name      string
+	Type      string
+	EnvCompat bool
+}
+
+var UpperCamelTransformer = &SecretsNameTransformer{
+	Name:      "Upper Camel",
+	Type:      "upper-camel",
+	EnvCompat: true,
+}
+var CamelTransformer = &SecretsNameTransformer{
+	Name:      "Camel",
+	Type:      "camel",
+	EnvCompat: true,
+}
+var DotNETTransformer = &SecretsNameTransformer{
+	Name:      ".NET",
+	Type:      "dotnet",
+	EnvCompat: false,
+}
```

Comment:
This transformer will produce keys that violate the env var spec. Should we exclude it from `run`?

Reply:
Absolutely 👍 The `run` command handler throws an error if you try to select a name transformer which has `EnvCompat` set to false. I just pushed another change which makes this clearer from the `--help` text.